test/steps/steps.js

```diff
@@ -7,12 +7,12 @@
 const Yadda = require('yadda');
 
 module.exports = (function() {
-    return Yadda.localisation.English.library()
-        .when('I visit $URL', url => {
+    return Yadda.localisation.English.library().
+        when('I visit $URL', url => {
             console.log("Visiting", url);
-        })
+        }).
 
-        .then('I should see "$STRING"', expected => {
+        then('I should see "$STRING"', expected => {
             console.log("Do I see", expected);
-        })
+        });
```

Comment:
Yeah, it doesn't have a rule for this. It's not per the style doc though. https://github.com/airbnb/javascript#18.5